readme.md

```diff
@@ -103,7 +103,7 @@ GitHub Enterprise is also supported: [How to enable it](https://fregante.github.
 <table>
 	<tr>
 		<th width="50%">
-			<p><a title="show-whitespace"></a> Makes whitespace characters visible
+			<p><a title="render-whitespace"></a> Makes whitespace characters visible
```

Comment:
Choosing the term used by popular editors like VSCode:

<img width="415" alt="image" src="https://user-images.githubusercontent.com/44045911/158072261-7d364f7e-fe35-4f95-941b-31138f1cfea7.png">